<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>David Baumr</author>
 <class>TranslatorConfigUi</class>
 <widget class="QWidget" name="TranslatorConfigUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>580</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="boxLanguage">
     <property name="title">
      <string>Source language settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_secondary">
          <property name="text">
           <string>Default source language</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_primary">
          <property name="text">
           <string>Alternative source language</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="primaryLanguage">
          <property name="toolTip">
           <string>Will be used if no source language is specified.</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="secondaryLanguage">
          <property name="toolTip">
           <string>Will be used if target language and default source language are identical.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boxAPI">
     <property name="title">
      <string>API settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_apikey">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;https://tech.yandex.com/keys/get/?service=trnsl&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;Yandex API key&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="yandexKey">
          <property name="toolTip">
           <string comment="kennung" extracomment="iaeiae">To translate sentences it is necessary to register at yandex.com and obtain an API key. If no valid key is entered, only single words can be translated.</string>
          </property>
          <property name="whatsThis">
           <string comment="iae" extracomment="aea"/>
          </property>
          <property name="placeholderText">
           <string comment="iaeuiaeuiae" extracomment="uiaeiaeuiae"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
